.editorconfig

```diff
@@ -38,13 +38,16 @@ indent_size = 2
 [*.yml]
 indent_size = 2
 
+[*.toml]
+indent_size = 2
```

Comment:
Please use taplo to do toml formatting

Reply:
This helps for IDE since most IDE doesn't identify taplo's config. We we indent_size = 2 in taplo also.